frontend/src/scss/settings/_uswds-theme-utilities.scss

```diff
@@ -36,7 +36,7 @@ $theme-utility-breakpoints: (
   // 240px:
   "card-lg": false,
   // 320px:
-  "mobile": false,
+  "mobile": true,
```

Comment:
I think we want to keep this off and the CSS that you're currently using `mobile` on, you don't use a media query. That keeps it mobile-first.

Reply:
Oh, I think now I catch what you mean.